### DISCUSSION TITLE
Render world copies in Flutter ?

### DISCUSSION THREAD

**Discussion by FabriceACohen** [maplibre/flutter-maplibre-gl#479] [Q&A]
Hello,

I try get my map to be able to scroll horizontally (and ideally vertically too) infinitally. 
As if I looping around the globe. Un web it seems to exist a setRenderWorldCopies method but I can't find something similar in Flutter. 
Any advice ?
Thanks,

[Doc :](https://maplibre.org/maplibre-gl-js/docs/examples/render-world-copies/)

**Comment by josxha** [comment]
Hi @FabriceACohen, theoretically it should be possible in the flutter package but unfortunally this isn't implemented.
Feel free to create a pull request if you want. (:
